src/components/App.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div id="app">
+    <template v-if="$store.state.user">
+      <h2>You are logged in!</h2>
+      <div class="logout">
+        <button @click="logout">Logout</button>
+      </div>
+      <router-view></router-view>
+    </template>
+    <template v-else>
+      <div class="login">
+        <Auth />
+      </div>
+    </template>
+  </div>
+</template>
+
+<script>
+import Auth from './Auth';
+
+export default {
```

Comment:
This declares "app" as a component to be used elsewhere

(From now on just assume I'm saying "I think". Assume I'm uncertain about all of this)